relay-auth/src/lib.rs

```diff
@@ -147,6 +147,17 @@ pub enum UnpackError {
     SignatureExpired,
 }
 
+/// Errors during the signature or verification process.
+#[derive(Debug, Clone, PartialEq, thiserror::Error)]
+pub enum RelaySignatureError {
```

Comment:
Any reason you're namespacing this error, it's already indirectly namespaced by the crate/module.